src/software/ai/hl/stp/tactic/goalie/goalie_tactic_test.py

```diff
@@ -3,23 +3,23 @@
 import pytest
 
 import software.python_bindings as tbots
-from proto.primitive_pb2 import MaxAllowedSpeedMode
-from proto.tactic_pb2 import AssignedTacticPlayControlParams, GoalieTactic, Tactic
 from software.simulated_tests.robot_enters_region import *
 from software.simulated_tests.ball_enters_region import *
 from software.simulated_tests.ball_moves_forward import *
 from software.simulated_tests.friendly_has_ball_possession import *
 from software.simulated_tests.ball_speed_threshold import *
 from software.simulated_tests.robot_speed_threshold import *
 from software.simulated_tests.excessive_dribbling import *
```

Comment:
Would be nice to have all these imports in one "validation function" import statement rather than manually importing every validation function. Not sure how easy that is to do though.

Reply:
i think this file is just a test dumping ground rn, perhaps we can make a validation folder so we can do 
~~~
from validation.excessive_dribbling import EventuallyStopExcessivelyDribbling
~~~

for example

Ill clean this up in my next PR when I do a pass on the simulated tests